docs/guide/deploy.md

```diff
@@ -260,6 +260,10 @@ Don't enable options like _Auto Minify_ for HTML code. It will remove comments f
    firebase deploy
    ```
 
+### Stormkit
```

Comment:
Can you move the section to bottom? The list is not alphabetical. We are keeping things in the order they were added to docs.